enos/provider/vbox.py

```diff
@@ -85,9 +89,10 @@ def init(self, config, calldir, force_deploy=False):
                             quiet_stderr=False)
         if force_deploy:
             v.destroy()
-        v.up()
+        # NOTE(matrohon) : force no parallel provisionning since it fails with
+        # libvrit/hostmanager plugins for vagrant
+        v._call_vagrant_command(['up', '--no-parallel'])
```

Comment:
Does it modify the deployment workflow, slowing down the deployment due to sequential provisioning of resources?